### PR TITLE
Update toolbox.py

### PR DESCRIPTION
**Pull request by AnkurArohi**
Updated multiple issues in toolbox.py 
Fix for https://github.com/e2nIEE/pandapower/issues/1592
Improved Pylint issues

### CODE REVIEW

File: pandapower/toolbox.py [hunk 1]
```diff
@@ -1,8 +1,13 @@
...
-
+"""
+changeset 19/5/2022
+function :  get_connected_elements
+error : element table queried before assingment
+reproduce : pp.get_connected_elements_dict(network , bus,
+respect_switches=False,respect_in_service=True)
+solution : query if there is the df before removing the in_service false
+elements
+"""
```
Comment (rbolgaryn) [inline pandapower/toolbox.py:10]: please remove this comment and bring back the license

File: pandapower/toolbox.py [hunk 2]
```diff
@@ -3240,143 +4148,11 @@ def false_elm_links_loop(net, elms=None):
...
+                fl = fl.union(
+                    false_elm_links(net, elm, "element", net[elm]["element_type"])
+                )
             else:
                 fl = fl.union(false_elm_links(net, elm, "bus", "bus"))
             if len(fl):
                 false_links[elm] = fl
     return false_links
-
-
```
Comment (rbolgaryn) [inline pandapower/toolbox.py:3250]: why are all of these functions removed?

**Comment by rbolgaryn** [comment]
tests fail, this PR is not ready for review yet

File: pandapower/toolbox.py (hunk 2 above)
Comment (AnkurArohi) [reply pandapower/toolbox.py:3250]: These will be added back.
Thanks for the Review

**Comment by AnkurArohi** [comment]
@rbolgaryn 
Can you please update this , thanks

**Comment by AnkurArohi** [comment]
relevant_nodes

> Hi @AnkurArohi, many thanks for contributing to pandapower! Besides the numerous Pylint changes, it is not easy to find the places where the functionality is really updated. @rbolgaryn the Pylint changes differ from previous code style of the pandapower package. I suggest, this PR should be split into functional updates and style changes. And regarding style changes, a pandapower issue should first discuss the need of further elaborated styles. Till now we are only with PeP8, but the changes of this PR seem to be much more, am I correct? The thing with more elaborated styles is, that there are many developer who (still?) do not consider that.

@SteffenMeinecke 
Yes @rbolgaryn had already updated me that the Pylint changes that I suggested were numerous and the difference between the bug fix and the other Pylint based changes is not easy to determine.
Unfortunately its the same at my end.
But, I am trying.
I can still ensure that the Pylint changes are infact not breaking any tests and Pandapower can only benefit from those.

**Comment by AnkurArohi** [comment]
Developed merged in patch

**Comment by AnkurArohi** [comment]
@SteffenMeinecke @rbolgaryn 
With the merge that i did pandapower will miss version based operations for packages which is not optimal solution.
But this is what pandapower has currently in develop 
There is no way of version based operation.
Please see the commit

**Comment by rbolgaryn** [comment]
Can you elaborate?

**Comment by rbolgaryn** [comment]
By the way I don't think we'll merge this pull request. I think it's best to open a new one where we only have that one line changed that fixes the bug + a test for the bug. This way it makes it easier to keep track of who did which change.

**Comment by AnkurArohi** [comment]
ok then i will close it completely 
All the best!!!!

**Comment by AnkurArohi** [comment]
> Can you elaborate?

image

see the version based operations

**Comment by rbolgaryn** [comment]
ignoring requests to change something won't get your pull requests merged

**Comment by AnkurArohi** [comment]
> ignoring requests to change something won't get your pull requests merged

well it will also then end up in pp having unresolved bugs, i see it to be both sided, I would have appreciated a collaborative work instead of request changes

**Pull request closed without merging**